### ISSUE TITLE
I get "java.net.SocketException: Connection reset by peer (connect failed)" error regularly

### ISSUE THREAD

**Issue by egor-kamenev** [Codeception/CodeceptJS#687]
I'm trying to run my tests and get "java.net.SocketException: Connection reset by peer (connect failed)" error regularly!

```bash
semrush.com --
 > [Session] Starting singleton browser session
 • I am on page "/" (3.498 sec)
 • I click "button[data-test="auth-popup__btn-login"]" (0.407 sec)
 • I fill field "input[data-test="auth-popup__email"]", "e.kamenev@semrush.com" (0.525 sec)
 • I fill field "input[data-test="auth-popup__password"]", "Fender1945" (0.19 sec)
 • I check option "input[data-test="auth-popup__remember"]" (0.143 sec)
 • I click "button[data-test="auth-popup__submit"]" (0.302 sec)
 • I am on page "/dashboard/" (8.819 sec)
   Within .lm-menu__content-wrapper:
   • Within .lm-menu__content-wrapper: I click {"css":"a[data-test=\"analytics\"][href=\"javascript:void(0);\"]"}
 > [SmartWait] Locating a[data-test="analytics"][href="javascript:void(0);"] in 15000
 (0.171 sec)
   • Within .lm-menu__content-wrapper: I click {"css":"a[data-test=\"domain_adsense\"]"}
 > [SmartWait] Locating a[data-test="domain_adsense"] in 15000
 (19.057 sec)
 is DA loaded
  • I am on page "/analytics/da/overview/" (6.296 sec)
  • I see element "div.asns-js-mediator"
 > [SmartWait] Locating div.asns-js-mediator in 15000
 (15.026 sec)
 > Screenshot has been saved to /Users/egor.kamenev/Desktop/da-integration-tests/output/is_DA_loaded.failed.png
 ✖ FAILED in 21962ms

 > [Session] cleaning cookies and localStorage
 ✖ "after each" hook: finialize codeceptjs for "is DA loaded" in 18ms
Error: java.net.SocketException: Connection reset by peer (connect failed)

-- FAILURES:

  1) semrush.com: is DA loaded:

      expected elements of div.asns-js-mediator to be seen
      + expected - actual

      -false
      +true
      
  
  Scenario Steps:
  
  - I.seeElement("div.asns-js-mediator") at Test.Scenario (tests/login/smoke_test.js:11:5)
  - I.amOnPage("/analytics/da/overview/") at Test.Scenario (tests/login/smoke_test.js:10:5)
  
  
  
  Run with --verbose flag to see NodeJS stacktrace

  2) semrush.com: /Users/egor.kamenev/Desktop/da-integration-tests/tests/login/smoke_test.js "after each" hook: finialize codeceptjs for "is DA loaded":
     java.net.SocketException: Connection reset by peer (connect failed)
  
  Run with --verbose flag to see NodeJS stacktrace


  FAIL  | 0 passed, 2 failed   // 56s

```
### Details

* CodeceptJS version: 1.1.0
* NodeJS Version: 6.11.0
* Operating System: macOs Sierra 10.12.6
* WebDriverIO
* Configuration file:

```js
{
  "output": "./output",
  "helpers": {
    "WebDriverIO" : {
      "url": "http://semrush.loc",
      "smartWait": 15000,
      "browser": "chrome",
      "restart": false,
      "windowSize": "maximize",
      "timeouts": {
        "script": 60000,
        "page load": 20000
      }
    }
  },
  "include": {
    "I": "./steps_file.js",
    "leftMenuFragment": "./fragments/LeftMenu.js",
    "emptyPage": "./pages/Empty.js",
    "overviewPage": "./pages/Overview.js",
    "headerFragment": "./fragments/Header.js"
  },
  "mocha": {},
  "bootstrap": false,
  "teardown": null,
  "hooks": [],
  "tests": "./tests/**/*_test.js",
  "timeout": 100000,
  "name": "da-integration-tests"
}
```

**Comment by APshenkin** [comment]
@egor-kamenev What version of selenium do you use? I found the same issue with 3.5.0 selenium. Looks like it's selenium issue, not Codeceptjs. Try 3.4.0 selenium and give us feedback about results.

Thanks!

**Comment by egor-kamenev** [comment]
@APshenkin Thank you! It works for me

**Comment by egor-kamenev** [comment]
@APshenkin Thank you!
It works for me:
```bash
selenium-standalone install --version=3.4.0
selenium-standalone start --version=3.4.0
```

**Comment by toien44** [comment]
Sorry this issue is closed but do you know if it's fixed in Selenium 3.5.3 I'm running into this issue too. It only happens only with 1 suite with `getText`.
